.github/workflows/int-and-connected-test-run.yml

```
@@ -108,100 +100,83 @@ jobs:
-          TDR_LOG_APPENDER: 'Console-Standard'
-        with:
-          actions_subcommand: 'gradletestrunnersmoketest'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Run integration tests via Gradle
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+      - name: Run integration tests
         env:
           # required for azure tests
           AZURE_CREDENTIALS_APPLICATIONID: 0e29ec36-04e8-44d5-ae7c-50dc15135571
           AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
           AZURE_CREDENTIALS_SECRET: ${{ secrets.AZURE_CREDENTIALS_SECRET }}
-          # unset the GitHub Action default JAVA_HOME to run with JDK 17
-          JAVA_HOME:
+          AZURE_SYNAPSE_SQLADMINUSER: ${{ secrets.AZURE_SYNAPSE_SQLADMINUSER }}
+          AZURE_SYNAPSE_SQLADMINPASSWORD: ${{ secrets.AZURE_SYNAPSE_SQLADMINPASSWORD }}
+          AZURE_SYNAPSE_WORKSPACENAME: tdr-snps-int-east-us-ondemand.sql.azuresynapse.net
+          # required for integration tests
+          GOOGLE_APPLICATION_CREDENTIALS: jade-dev-account.json
+          GOOGLE_SA_CERT: jade-dev-account.pem
+          IT_JADE_API_URL: http://localhost:8080
+          # postgres connection details
+          PGHOST: 127.0.0.1
+          PGPASSWORD: postgres
+          # required for integration tests
+          RBS_CLIENT_CREDENTIAL_FILE_PATH: rbs-tools-sa.json
+          RBS_INSTANCEURL: https://buffer.tools.integ.envs.broadinstitute.org
+          RBS_POOLID: datarepo_v1
           # output plain logs instead of json
           TDR_LOG_APPENDER: 'Console-Standard'
-        with:
-          actions_subcommand: 'gradleinttest'
-          pgport: ${{ job.services.postgres.ports[5432] }}
-          test_to_run: 'testIntegration'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Clean state lock from used Namespace on API deploy
-        if: always()
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        with:
-          actions_subcommand: 'k8_checknamespace_clean'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Clean whitelisted Runner IP
+        run: |
+          # wait for postgres to be ready
+          pg_isready -h ${PGHOST} -t 30
+          # create the datarepo and stairway databases
+          psql -U postgres -f ./db/create-data-repo-db
+          # extract service account credentials
+          base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
+          jq -r .private_key ${GOOGLE_APPLICATION_CREDENTIALS} > ${GOOGLE_SA_CERT}
+          chmod 644 ${GOOGLE_SA_CERT}
+          base64 --decode <<< ${{ secrets.B64_RBS_APPLICATION_CREDENTIALS }} > ${RBS_CLIENT_CREDENTIAL_FILE_PATH}
+          # build code and run local api
+          ./gradlew bootJar
+          export DATA_REPO_JAR=$(find . -type f -name jade-data-repo-*-SNAPSHOT.jar)
+          java -jar ${DATA_REPO_JAR} > local-api-output.log &
```

Looking at this again, I'm wondering why we run out of a jar instead of using `./gradlew bootRun &`.
```suggestion
          ./gradlew bootRun > local-api-output.log &
```

---

`./gradlew` starts up multiple runners, so a redirect will lead to an empty file.